action/ExecutePath.action

```diff
@@ -0,0 +1,18 @@
+# The trajectory to execute
+RobotTrajectory trajectory
+
+---
+
+# Error code - encodes the overall reason for failure
+MoveItErrorCodes error_code
+
+# The full starting state of the robot at the start of the trajectory
+moveit_msgs/RobotState trajectory_start
+
+# The trace of the trajectory recorded during execution
```

Comment:
could you explain "trace" better... in what cases would it differ from the input ``trajectory`` and at what discretization level are you expecting this returned trajectory to be at?